Sources/CorePayments/Networking/GraphQL/GraphQLQueryResponse.swift

```diff
@@ -1,4 +1,4 @@
-struct GraphQLQueryResponse<T: Codable>: Codable {
+public struct GraphQLQueryResponse<T: Codable>: Codable {
```

Comment:
Same comment re: `nodoc`

Reply:
Addressed in 8a67bce871356c3a8bd910be35cc7efd64d1864c